.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,89 @@
+  name: "🐛 Bug Report"
+  description: "Submit a bug report to help us improve"
+  title: "[Bug]: "
+  labels: ["bug"]
+  body:
+    - type: markdown
+      attributes:
+        value: We value your time and your efforts to submit this bug report is appreciated. 🙏
+
+    - type: textarea
+      id: description
+      validations:
+        required: true
+      attributes:
+        label: "📜 Description"
+        description: "A clear and concise description of what the bug is."
+        placeholder: "It bugs out when ..."
+
+    - type: textarea
+      id: steps-to-reproduce
+      validations:
+        required: false
+      attributes:
+        label: "👟 Reproduction steps"
+        description: "How do you trigger this bug? Please walk us through it step by step."
+        placeholder: "1. Go to '...'
+          2. Click on '....'
+          3. Scroll down to '....'
+          4. See error"
+
+    - type: dropdown
+      id: operating-system
+      attributes:
+        label: "💻 Operating system"
+        description: "What OS is your app running on?"
+        options:
+          - Linux
+          - MacOS
+          - Windows
+          - Something else
+      validations:
+        required: true
+
+    - type: textarea
+      id: additional-context
+      validations:
+        required: false
+      attributes:
+        label: "📃 Provide any additional context for the Bug."
+        description: "Add any other context about the problem here."
+        placeholder: "It actually ..."
+
+    - type: textarea
+      id: logs
+      validations:
+        required: false
+      attributes:
+        label: 📖 Relevant log output
+        description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+        render: shell
+
+    - type: checkboxes
+      id: no-duplicate-issues
+      attributes:
+        label: "👀 Have you spent some time to check if this bug has been raised before?"
+        options:
+          - label: "I checked and didn't find similar issue"
+            required: true
```

Comment:
This doesn't have to be mandatory. We are happy to get multiple reports about the same bug.